bin/parse_viral_pred.py

```diff
@@ -235,13 +235,16 @@ def parse_virus_sorter2(sorter_files, vs_cutoff):
     
     boundary_df = pd.read_csv(final_boundary_file, sep='\t', index_col='seqname', usecols=boundary_columns, 
                               dtype=boundary_dtypes)
+
```

Comment:
no need to do 2 lines. those should be between functions according to PEP8. Leave 1 line separation